src/routes/formsgSiteCreation.ts

```diff
@@ -119,6 +119,13 @@ export class FormsgRouter {
         repoName,
         isEmailLogin,
       })
+
+      if (isEmailLogin) {
+        // NOTE: Clone site to EFS
+        const cloneRes = await this.gitFileSystemService.clone(repoName)
+        if (cloneRes.isErr()) throw cloneRes.error
```

Comment:
should we add an alarm for this error? else we won't know right

Reply:
outer loop has a `catch` that does `sendCreateError`